### DISCUSSION TITLE
2009/11/29/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%8B%98%E8%AF%AF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#285] [Announcements]
# 2009/11/29/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%8B%98%E8%AF%AF

营养药和美容护肤品

https://blog.qizhen.xyz/2009/11/29/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%8B%98%E8%AF%AF

**Comment by ruanqizhen** [comment]
之前博客下的留言：

xiaohei
2011年08月25日 23:41 编辑
看来我看书不够仔细，我看到快200页了，只发现85页的错处

回复
Qizhen
2009年11月30日 12:33 编辑
今天要特别感谢 Adeline！我今天更新的内容都是Adeline帮我指出的，非常钦佩她的认真态度。

回复
Qizhen
2009年11月4日 14:50 编辑
谢谢你的反馈，我会把修正加入勘误表。

回复
Yundong
2009年11月4日 10:33 编辑
给个反馈，书中图11.19和图11.20完全一样。网上的和描述的一致。

回复
帆
2009年09月17日 23:14 编辑
赞作者的负责！

回复
